docs/troubleshooting-guide.md

```diff
@@ -0,0 +1,146 @@
+```
+lerna ERR! npm install exited 1 in '@zooniverse/react-components'
+lerna ERR! npm install stderr:
+npm ERR! code E404
+npm ERR! 404 Not Found: @zooniverse/grommet-theme@2.0.0
+```
+
+Analysis:
+- This happens because Package A (react-components) requires Package B
+  (grommet-theme), and in this case, Package B needs to be **built** first
+  before Package A can use it.
+
+Solution:
+- Build Package B first, e.g.: `cd packages/grommet-theme ; npm install ; npm run build`
+- Run `lerna link` to 'register' the newly built package.
+- _Optional/untested: Then run `lerna bootstrap` again._
+
+
+**Issue 2:** can't run installed dependency
+
+- Occurs when running `npm run dev` or similar script from within a package.
+
+Example error message:
+```
+lib-classifier$ npm run _check
+
+> @zooniverse/classifier@0.0.1 _check /Users/shaun/projects/front-end-monorepo/packages/lib-classifier
+> check-dependencies
+
+sh: check-dependencies: command not found
+npm ERR! file sh
+npm ERR! code ELIFECYCLE
+npm ERR! errno ENOENT
+npm ERR! syscall spawn
+npm ERR! @zooniverse/classifier@0.0.1 _check: `check-dependencies`
+npm ERR! spawn ENOENT
+npm ERR! 
+npm ERR! Failed at the @zooniverse/classifier@0.0.1 _check script.
+```
+
+Analysis
+- This is a bizarre error that occurs when npm scripts can't run certain
+  dependencies, despite the fact that the dependency has been downloaded.
+  (e.g. by running `npm ls check-dependencies` or looking inside the package's
+  node_modules folder.)
+- There's 
+
+Solution:
+- Remove the package's dependencies and reinstall them via bootstrap.
+- Remove the package's node_modules: `cd packages/your-package; rm -fr node_modules`
+- Re-run bootstrap: `lerna bootstrap`
+
+
+**Issue 3:** inexplicable error messages
+
+Analysis:
+- You ran into a general problem. It... it happens. Cover your bases by running
+  the following _"I dunno but let's see what happens"_ solutions!
+
+Solutions:
+- Did you install `lerna` globally? Install it by running `npm install -g lerna`
+- Did you update `lerna` recently? Update it by running `npm install -g lerna` (Sanity check: `lerna 3.4.3` works for the monorepo's Oct 2018 build)
+- Maybe when you ran `lerna bootstrap`, it's possible that something went wrong and **packages that are supposed to be dependencies of each other aren't symlinked properly.**
+  - To force this, run `lerna link` before running `lerna bootstrap`
+
+## War Stories
+
+Installing the monorepo will likely lead to different experiences for everyone.
+
+The following tracks Shaun's 8th attempt.
+
+```
+$ git clone git@github.com:zooniverse/front-end-monorepo.git
+$ cd front-end-monorepo
+$ lerna bootstrap
+=> ERROR: in zooniverse/react-components, 404 Not Found zooniverse/grommet-theme@2.0.0
+
+$ cd packages/lib-grommet-theme
+$ npm install
+$ npm run build
+$ lerna link
+//At this point we can already run app-project. i.e. cd ../app-project ; npm run dev
+=> SUCCESS!
+
+$ cd ../lib-react-components/
+$ npm install
```

Comment:
`lerna bootstrap --scope=@zooniverse/react-components` should be run instead here since this package depends on the symlinked lib-grommet-theme.

Reply:
Noted on needing to try `--scope`, though I'll add my notes on that to my 9th Iteration War Stories instead of modifying the 8th Iteration War Stories with the corrected actions.

I can't fudge the details of what already happened, otherwise it'd be Fake Documentation. Sad!